opentelemetry-sdk/Cargo.toml

```diff
@@ -28,7 +28,7 @@ url = { workspace = true, optional = true }
 tokio = { workspace = true, features = ["rt", "time"], optional = true }
 tokio-stream = { workspace = true, optional = true }
 http = { workspace = true, optional = true }
-tracing = {workspace = true, optional = true}
+tracing = {workspace = true}
```

Comment:
@lalitb Looking at this again, I think we should be do a feature-flag, and make it part of default. If someone really wants to avoid `tracing` dependency, they can do so by removing feature flags.
(Tracing causes assembly size increase due to statics, so the more callsites we have, the more the size.. some people may avoid to avoid that...)
Thoughts?
ff can be "internal-logs" (without the experimental)

Reply:
This is done.

Reply:
Just realized that this will also make the errors and warnings optional depending on the flag. Will update the macros to dump them using `eprintln` if the flag is disabled.

Reply:
Have added eprintln for errors and warning if the flag is disabled. Please have a look, if there is any concern.

Reply:
I think its best to do that along with removal of current global error handler.

Reply:
Make sense. done.